app/guid-node/forks/template.hbs

```diff
@@ -0,0 +1,49 @@
+{{title (t 'forks.page_title' nodeTitle=model.taskInstance.value.title)}}
+<div class="container">
+    {{#bs-modal open=newModal onSubmit=(action 'newFork') as |m|}}
+        {{#m.body}}
+            <h3>{{t 'forks.create_fork_modal'}}</h3>
+        {{/m.body}}
+        {{#m.footer}}
+            {{#bs-button onClick=(action (mut newModal) false)}}{{t 'general.cancel'}}{{/bs-button}}
+            {{#bs-button onClick=(action m.submit) type='info'}}{{t 'forks.fork'}}{{/bs-button}}
+        {{/m.footer}}
+    {{/bs-modal}}
+    {{delete-node-modal delete=(action 'delete') openModal=deleteModal closeModal=(action 'closeDeleteModal') nodeType=(t 'general.project')}}
```

Comment:
No, nodeType is either a project or component, while category can be things like "Software" or "Data". Forks specifically seem to be only projects. Will double check and implement this properly.

Reply:
In this particular case, yes, nodes have to be projects. More specifically, though, if `node_id == root_id` then it's a project, otherwise it's a component. That will be useful when we start using this for other things.

Reply:
ooh, `isRoot` would be a useful computed property on the node model.